### PR TITLE
[MB-6419] update prime demo script to add DDFSIT to the flow

### PR DESCRIPTION
**Pull request by pearl-truss**
## Description

This PR introduces a few changes so that the prime api demo script follows an updated flow to match and highlight our current feature work
 - it adds a DDFSIT service item into the create payment request payload
 - it updates the edi generator to be able to handle a DDFSIT service item
 - it updates the default proof of services to expect a single pdf document instead of a pdf, png and jpg 
 - it creates a pause before submitting the payment request so that the TOO can first approve the service items
 - it adds ability to create multiple payment request

## Setup
add a pdf called `proof_of_service.pdf` to `tmp/uploads`

```sh
make db_dev_e2e_populate && make server_run
prime-api-demo 9c7b255c-2981-4bf8-839f-61c7458e2b4d
```

## Acceptance
Run in staging and use
`prime-api-demo <mtoID> api.stg.move.mil`

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-6419) for this change

### CODE REVIEW

File: scripts/prime-api-demo [hunk 1]
```diff
@@ -11,15 +11,15 @@ function usage() {
   echo
   echo "If omitted, the optional parameters are set as follows:"
   echo "  hostname = local"
-  echo "  proofs of service = (./tmp/uploads/proof_of_service.png ./tmp/uploads/proof_of_service.jpg ./tmp/uploads/proof_of_service.pdf)"
+  echo "  proofs of service = (./tmp/uploads/proof_of_service.pdf)"
```
[the remark this answers is not in the document]
Comment (pearl-truss) [reply scripts/prime-api-demo:14]: yeah by default it expects just 1 proof of service now

File: scripts/prime-api-demo [hunk 2]
```diff
@@ -190,67 +225,19 @@ cat > tmp/pad_create_ddfsit_service_item.json <<- EOM
...
-for proofOfService in "${proofs[@]}"
-do
-  echo upload "$proofOfService"
-  proofBase=$(basename "${proofOfService}")
-  bin/prime-api-client "${primeapiopts[@]}" create-upload --paymentRequestID "${prID}" --filename "${proofOfService}" > tmp/pad_create_upload_response_"${proofBase}".json
-
-  jq . tmp/pad_create_upload_response_"${proofBase}".json
 
-  echo
+createPaymentRequest
```
Comment (gidjin) [inline scripts/prime-api-demo:235]: Might be good to add parens just to make it clear that this is a function call. Assuming that is syntactically allowed.

**Review by gidjin — APPROVED** [review]
Changes look good, script worked as expected locally. One minor nit comment.

File: scripts/prime-api-demo (hunk 2 above)
Comment (pearl-truss) [reply scripts/prime-api-demo:235]: I agree parens would make it clearer that it's a function but adding them caused a syntax error.

**Pull request merged**